dev-libs/libaddressinput/libaddressinput-0~git.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="Google’s postal address library, powering Android and Chromium"
+DESCRIPTION="libaddressinput can provide information about \
+what input fields are required for a correct address input form for \
+any country in the world and can validate an address to highlight \
+input errors like missing required fields or invalid values."
+HOMEPAGE="https://github.com/googlei18n/libaddressinput"
+COPYRIGHT="2014 Google Inc."
+LICENSE="Apache v2"
+REVISION="1"
+srcGitRev="a575c63de328d0c72237c55d1a05e191c245122a"
+SOURCE_URI="https://github.com/googlei18n/libaddressinput/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="f92b38a0291372aa97306626f2941485bd1c00bd3cc7e242a1cb9fcd50b62212"
+SOURCE_FILENAME="libaddressinput-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="libaddressinput-$srcGitRev"
+
+ARCHITECTURES="?x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+PROVIDES="
+	libaddressinput$secondaryArchSuffix = $portVersion
+	lib:libaddressinput$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	re2
+	"
+
+PROVIDES_devel="
+	libaddressinput${secondaryArchSuffix}_devel
+	devel:libaddressinput$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	libaddressinput$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	gtest_devel
+	re2_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:c++
+	cmd:git
+	cmd:wget
+	cmd:ninja
+	cmd:gyp
+	"
+
+BUILD()
+{
+	# Download missing packages
+	cd externals
+	rm -rf grit
+	git clone https://chromium.googlesource.com/chromium/src/tools/grit
+	rm -rf rapidjson
+	git clone https://github.com/Tencent/rapidjson
+	rm -rf googletest
+	rm -f release-1.8.0.tar.gz
+	wget https://github.com/google/googletest/archive/release-1.8.0.tar.gz
```

Comment:
Neither BUILD nor INSTALL should download anything. They should user SOURCE_URI_2, SOURCE_URI_3, and so on, as well as CHECKSUM_SHA256_2, CHECKSUM_SHA256_3, and so on.